mne/minimum_norm/resolution_matrix.py

```diff
@@ -116,20 +325,64 @@ def get_point_spread(resmat, src, idx, norm=False):
         Forward Operator.
     src : instance of SourceSpaces
         Source space used to compute resolution matrix.
-    idx : list of int
-        Vertex indices for which PSFs or CTFs to produce.
-    norm : bool
-        Whether to normalise to maximum across all PSFs (default: False).
+    idx : list of int | list of Label
+        Source for indices for which to compute PSFs or CTFs. If mode is None,
+        PSFs/CTFs will be returned for all indices. If mode is not None, the
+        corresponding summary measure will be computed across all PSFs/CTFs
+        available from idx.
+        Can be:
+
+        * list of integers : Compute PSFs/CTFs for all indices to source space
+          vertices specified in idx.
+        * list of Label : Compute PSFs/CTFs for source space vertices in
+          specified labels.
+
+    mode : None | 'mean' | 'max' | 'svd'
```

Comment:
This is very close to our `extract_label_time_course` API. This makes me think:

1. It's important to add a `Notes` saying why it differs (if at all)
2. Why can't we just use `extract_label_time_course(psf, labels, mode='max')`? I guess one reason is that we can't get `n_comp`, we're stuck with 1 in `extract_label_time_course`. So I can live with them being separate. Will the output be the same when `n_comp=1`? If so, please add a test verifying it.
3. We should probably rename `svd` to `pca` here for API consistency.
4. `flip` variants should be added -- the PSF/CTF are signed so flipping makes sense

Reply:
1 and 2: I hadn't thought of the similarity to extract_label_time_course. 
One different is the n_comp option, as you say. 
I don't think "flip" is recommended for PSF/CTFs, as it's important to know how contributions from different vertices cancel out in the mean/sum/pca. For example, it would be informative to know if the average point-spread from one label to another one is zero. This should be the default. Can we add the 'flip' later (if really necessary)?
In extract_label_time_courses, flipping and averaging is done over time courses. I would need to do this across vertices (i.e. flip every PSF/CTF)? The time axis is not meaningful for PSFs/CTFs, which I think is a reason to have separate functions (plus above).
3. Will rename svd to pca.
4. See response to '1 and 2'.

Reply:
> For example, it would be informative to know if the average point-spread from one label to another one is zero. This should be the default. Can we add the 'flip' later (if really necessary)?
In extract_label_time_courses, flipping and averaging is done over time courses. I would need to do this across vertices (i.e. flip every PSF/CTF)?

Yes but keep in mind `mean_flip` just flips the signs of the vertex values (time courses or PSF/CTF), and it's completely spatial -- it does not vary as a function of "time" or anything else. It's totally a property of the source space. `'mean_flip'` in `extract_label_time_course` (and here) is (would be) functionally equivalent to just having changed your `forward['source_nn']`, i.e., the source normal, to point in the opposite direction and doing your inverse using that, then doing the standard mean over whatever values you have in the label (time or PSF/CTF). To me this mode is especially useful because we recommend doing `mean_flip` over labels instead of `mean` in `extract_label_time_course` to combat this cancellation problem. Hence why I'm pushing for it.

But someone motivated to try it can always add it later :)

Reply:
I agree it makes total sense for activation time courses. However, the fact that it varies over vertices but not over time is a problem for PSFs/CTFs. It might make sense to "flip" individual PSFs/CTFs (e.g. such that they all correlated positively with each other), but not vertices within PSFs/CTFs. I'll need to think about this more.
I'm happy to add this later if required/desired, but for now I'll move on with the options that are already there.

Reply:
Sounds good